homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,264 +76,284 @@
+        sensor_id=SENSOR_HUMIDITY_SUPPLY,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_SUPPLY_FAN_SPEED,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Supply fan speed",
+        unit_of_measurement="rpm",
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_SPEED,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_SUPPLY_FAN_DUTY,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Supply fan duty",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_DUTY,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_FAN_SPEED,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust fan speed",
+        unit_of_measurement="rpm",
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_SPEED,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_FAN_DUTY,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust fan duty",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_DUTY,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust temperature",
+        unit_of_measurement=TEMP_CELSIUS,
+        sensor_id=SENSOR_TEMPERATURE_EXHAUST,
+        multiplier=0.1,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_EXHAUST_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust humidity",
+        unit_of_measurement=PERCENTAGE,
+        sensor_id=SENSOR_HUMIDITY_EXHAUST,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_SUPPLY,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Supply airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-plus",
+        sensor_id=SENSOR_FAN_SUPPLY_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_AIR_FLOW_EXHAUST,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Exhaust airflow",
+        unit_of_measurement=VOLUME_FLOW_RATE_CUBIC_METERS_PER_HOUR,
+        icon="mdi:fan-minus",
+        sensor_id=SENSOR_FAN_EXHAUST_FLOW,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_BYPASS_STATE,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        name="Bypass state",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:camera-iris",
+        sensor_id=SENSOR_BYPASS_STATE,
+    ),
+    ComfoconnectSensorEntityDescription(
+        key=ATTR_DAYS_TO_REPLACE_FILTER,
+        device_class=None,
```

Comment:
```suggestion
        device_class=None,
```
Don't define default values